src/main/scala/org/constellation/invertedmap/Inverted.scala

```diff
@@ -0,0 +1,40 @@
+package org.constellation.invertedmap
+
+object Inverted {
+  def isEmpty(m: Map[_, _]): Boolean = m.isEmpty
+
+  def contains[A](m: Map[_, Set[A]], x: A): Boolean =
+    m.exists { case (_, v) => v.contains(x) }
+
+  def get[A, B](m: Map[B, Set[A]], x: A): Option[B] =
+    m.find { case (_, v) => v.contains(x) }.map { case (k, _) => k }
+
+  def update[A, B, B1 >: B](m: Map[B1, Set[A]], k: A, v: B): Map[B1, Set[A]] = {
```

Comment:
Shouldn't it be the other way round?
`k: B, v: A`
Maybe using K, V instead of A, B would be better altogether?

Reply:
no, it has correct typing

Reply:
I wanted to avoid using `K` and `V` because it can be misleading what is a key and what is not